src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1948,7 +1949,13 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
                         }
                         isVisibleResponse = false;
                     }
-
+                }
+                
+                if (question.questionType == FeedbackQuestionType.CONTRIB) {
```

Comment:
Currently, in instructorfeedbackresults, sections are still displayed to instructors who don't have section privileges to view the responses/comments. However they won't be shown the responses/comments (shown below). 

![](https://slack-files.com/files-tmb/T054U3FEJ-F09H2STMY-21460d4ab4/section_1024.png)
If I do consider section privileges for `rosterSectionTeamNameTable` and not include the sections which the instructor does not have permissions for, then the sections panels won't be displayed. 

To avoid this, perhaps I could add a `sectionTeamNameTable` in the bundle which basically stores what `rosterSectionTeamNameTable` stores, but does not include sections which the instructor does not have privileges for?